src/components/AddRecipe.tsx

```diff
@@ -15,57 +16,44 @@ export interface RecipeFormState {
 type HTMLFormControls = HTMLInputElement | HTMLSelectElement | HTMLTextAreaElement;
 
 export const AddRecipe: FC<AddRecipeProps> = (props) => {
-
-  const [ recipeForm, setRecipeForm ] = useState<RecipeFormState> ({
-    title: "",
-    ingredients: "",
-    instructions: "",
-  })
+  const [recipeForm, setRecipeForm] = useState<RecipeFormState>({
+    title: '',
+    ingredients: '',
+    instructions: '',
```

Comment:
When do you use single quotes vs double quotes?

Reply:
Prettier made this change for me :)